man/man8/flow_orifice.8

```diff
@@ -0,0 +1,59 @@
+.TH flow_orifice 8  "2017-04-30" "USER COMMANDS"
+.SH NAME
+flow_orifice \- Trace flows for their orifices during traversing the network stack. Uses Linux eBPF/bcc.
+.SH SYNOPSIS
+.B flow_orifice [\-h] [\-i]
+.SH DESCRIPTION
+This traces all flows regardless of their protocol, showing address, port,
+protocol, the qdisc index and the driving CPU behind the network stack
```

Comment:
there is no such thing as 'qdisc index' in linux kernel.
It seems the tool dumps skb->queue_mapping and it suppose to compare it with cpu doing xmit?
What does it tell users?